app/src/main/java/com/simplecity/amp_library/playback/MusicService.java

```diff
@@ -1528,8 +1535,8 @@ public void open(List<Song> songs, final int position) {
     /**
      * Moves the item at index1 to index2.
      *
-     * @param from
-     * @param to
+     * @param from index1
```

Comment:
I don't understand this change - the param names are `from` and `to`, not `index1` and `index2`